File: 1012-Area/1012.go
package main

import (
	"fmt"
	"math"
)

func main() {

	var a, b, c float64
	var triangle, circle, trapezium, square, rectangle float64
	var pi = 3.14159
	fmt.Scan(&a, &b, &c)

	triangle = (a * c) / 2.0
	circle = pi * math.Pow(c, 2)
	trapezium = (a + b) / 2.0 * c
	square = math.Pow(b, 2)
	rectangle = a * b

	fmt.Printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n\n", triangle, circle, trapezium, square, rectangle)

}

File: 1012-Area/1012.py
import math
a, b, c = map(float, input().split())

pi = 3.14159
triangle = (a * c) / 2.0
circle = pi * math.pow(c, 2)
trapezium = (a + b) / 2.0 * c
square = math.pow(b, 2)
rectangle = a * b

print("TRIANGULO: {:.3f}\nCIRCULO: {:.3f}\nTRAPEZIO: {:.3f}\nQUADRADO: {:.3f}\nRETANGULO: {:.3f}".format( triangle, circle, trapezium, square, rectangle))
